net-analyzer/linkchecker/linkchecker-10.0.0.ebuild

```
@@ -0,0 +1,58 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
```

```suggestion
PYTHON_COMPAT=( python3_{7..9} )
```

python 3.6 support has been dropped

---

Python 3.6 is still in portage, if anyone wishes to use it, it's possible. Otherwise if you run a current system then you'll only have python3.8 anyway.